README.md

```diff
@@ -1,4 +1,4 @@
-# juice-juicehoue
+# juice-juicehouse
```

Comment:
This should actually be juice-interface now :)

Reply:
@odd-amphora Changed it, thanks!